templates/statsd_mapping.conf.erb

```diff
@@ -0,0 +1,8 @@
+<% @real_statsd_maps.each do |metric| -%>
```

Comment:
Can you change this to `real_statsd_maps.sort.each` ? 
Old Ruby versions have a bug where the order is not deterministic and it triggers the service restart on each run even if the data didn't change.